LayoutTests/fast/dom/css-element-attribute-js-null.html

```diff
@@ -71,6 +73,13 @@
                          {name: 'selectorText', expectedNull: 'null'}
                     ]
                 },
+                {
+                    type: 'CSSPageRule',
+                    elementToUse: pageRule,
+                    attributes: [
+                         {name: 'selectorText', expectedNull: '@page null'}
```

Comment:
Serializing a missing selector as the literal string null is almost certainly wrong. I recommend adding web-platform-tests coverage first.

Reply:
I am planning to merge this next - https://chromium.googlesource.com/chromium/src/+/2947176ac96b969ae7e0f119cc3e082c115d4e4e, which might address your concern.

Reply:
I'm not sure, that also seems to have the weird serialization of ending up with null. Although https://software.hixie.ch/utilities/js/live-dom-viewer/?%3Cstyle%3E%40page%20%7B%7D%3C%2Fstyle%3E%0A%3Cscript%3Ew(document.styleSheets%5B0%5D.cssRules%5B0%5D.selectorText)%3C%2Fscript%3E suggests they do end up with the empty string so maybe the test harness is just bad. Again having web-platform-tests coverage would make that more apparent.